src/scss/includes/direction-form.scss

```diff
@@ -32,19 +32,23 @@
 
     .direction-form-separator {
       border-top: 1px solid #c8cbd3;
-      margin: 0 37px;
+      margin: 0 52px 0 44px;
```

Comment:
we might try to create a flex layout for the form elements later

Reply:
Isn't it possible right now?

Reply:
I would like to push this change and the font changes before, just to have the general visual elements in place before changing layout/DOM :)

Reply:
I just added minor font changes